docs/kb/serve.md

```diff
@@ -33,7 +33,7 @@ The `ignite chain serve` command performs the following tasks:
 - Installs dependencies
 - Imports state, if possible
 - Builds protocol buffer files
-- Optionally generates JavaScript (JS), TypeScript (TS), and Vuex clients
+- Optionally generates TypeScript (TS) clients
```

Comment:
what happened to JS? no more Vuex?

Reply:
@marinhoarthur  I am very willing to learn what we need to document for our users, teach me!

Reply:
We only generate a TypeScript client, no JS, no Vuex.